fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1224,6 +1233,11 @@ create_stmt ::=
     {:
         RESULT = new CreateDbStmt(ifNotExists, db, null);
     :}
+    /* Datasource */
+    | KW_CREATE KW_EXTERNAL KW_DATASOURCE ident:ds opt_properties:properties
+    {:
+         RESULT = new CreateDatasourceStmt(ds, properties);
+    :}
```

Comment:
And please add a config in `Config.java` like `enable_multi_catalog`, and default is `false`,
to forbid this under-developement feature